app/models/mixins/miq_ae_set_user_info_mixin.rb

```diff
@@ -1,7 +1,7 @@
 module MiqAeSetUserInfoMixin
   extend ActiveSupport::Concern
   included do
-    before_validation  :set_user_info
+    before_validation :set_user_info, :if => :changed?
```

Comment:
are there actual fields that we care about?

can we trim the target down?
if it is 2 fields, I have a few ideas

Reply:
we have similar need for "if changed" logic elsewhere.... @NickLaMuro was working on a similar one recently.

Reply:
@kbrock yeah, it's two fields